src/app/components/shared/menu/link/link.component.scss

```diff
@@ -1,3 +1,8 @@
+.disabled {
+  cursor: not-allowed;
+  opacity: 0.75;
+}
+
```

Comment:
bad indentation?

~~does this style compund with the .button:disabled style I saw before?~~

I think this is a good example for what I meant by refactoring the .button:disabled style.